src/network.js

```diff
@@ -171,6 +171,36 @@ function postComment(parentId, markdownBody) {
   post('/api/comment', { 'parent' : parentId, 'text' : markdownBody });
 }
 
+function getComment(commentId) {
+  const comment = get('https://www.reddit.com/api/info.json?id=' + commentId);
+
+  if (comment.length == 0) return;
+
+  const data = comment[0]['data'];
+
+  return {
+    'body': data['body'],
+    'author': data['author'],
+    'id': data['name'],
+    'link_id' : data['link_id'],
+    'postTitle': 'This is a dummy value.',
```

Comment:
`This is a dummy value.`? Does it not return the post title?

Reply:
The reddit API is, unfortunately, inconsistent with the values it returns. The comment end point does not return a value, but the project requires a `postTitle` value so I added the dummy value.

https://www.reddit.com/api/info.json?id=t1_do2yo6a

Reply:
I agree, no need to make an extra network call for this. 

Maybe we can change it to an empty string and add a comment beside it "Not returned by reddit endpoint"? When I first read it, I thought it might have been some accidental test code left in

Reply:
Very good point, done! :-)